database/seeders/ReleaseTablesSeeder.php

```diff
@@ -15,11 +15,11 @@ public function run(): void
     {
         $releasesData = getReleasesFromFile() ?? [];
 
-        if (!empty($releasesData['integration']) !== null && IntegrationRelease::count() === 0) {
+        if (array_key_exists('integration', $releasesData) && !empty($releasesData['integration']) !== null && IntegrationRelease::count() === 0) {
```

Comment:
The same change needs to be applied to the right. It seems illogical to check both `!empty` and `!== null`, as you've corrected for the front half of the condition.

Reply:
Good catch, I've updated the second part to remove the `!== null` conditional. Thanks for reviewing!